[Bug Report] types error

<!-- generated by https://elementui.github.io/issue-generator DO NOT REMOVE -->

### Element Plus version
1.2.0-beta.3

### OS/Browsers version
mac 12

### Vue version
3.2.22

### Reproduction Link
https://github.com/element-plus/element-plus/

### Steps to reproduce
``` sh
npx vue-tsc --noEmit
```

### What is Expected?
no error

### What is actually happening?
```
node_modules/element-plus/es/components/index.d.ts:1:15 - error TS2307: Cannot find module './affix' or its corresponding type declarations.

1 export * from './affix';
                ~~~~~~~~~

node_modules/element-plus/es/components/index.d.ts:51:15 - error TS2307: Cannot find module './slider' or its corresponding type declarations.

51 export * from './slider';
                 ~~~~~~~~~~

node_modules/element-plus/es/tokens/config-provider.d.ts:1:42 - error TS2307: Cannot find module 'element-plus/es/components/config-provider/config-provider' or its corresponding type declarations.

1 import type { configProviderProps } from 'element-plus/es/components/config-provider/config-provider';
                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

<!-- generated by https://elementui.github.io/issue-generator DO NOT REMOVE -->

--- a/package.json
+++ b/package.json
@@ -82,7 +82,7 @@
     "@types/sass": "1.43.1",
     "@typescript-eslint/eslint-plugin": "5.6.0",
     "@typescript-eslint/parser": "5.6.0",
-    "@vue/compiler-sfc": "3.2.23",
+    "@vue/compiler-sfc": "3.2.24",
     "@vue/test-utils": "2.0.0-rc.16",
     "chalk": "4.1.2",
     "components-helper": "1.0.4",
@@ -114,7 +114,7 @@
     "ts-jest": "26.5.6",
     "ts-morph": "13.0.2",
     "typescript": "4.5.2",
-    "vue": "3.2.23",
+    "vue": "3.2.24",
     "vue-jest": "5.0.0-alpha.10",
     "vue-router": "4.0.12",
     "vue-tsc": "0.29.8"

--- a/package.json
+++ b/package.json
@@ -86,6 +86,7 @@
     "@vue/test-utils": "2.0.0-rc.16",
     "chalk": "4.1.2",
     "components-helper": "1.0.4",
+    "csstype": "2.6.19",
     "cz-conventional-changelog": "3.3.0",
     "esbuild": "0.14.2",
     "eslint": "8.4.0",

